index.js

```diff
@@ -1,5 +1,31 @@
 function scoreCalculator(arr) {
-  // do work here
+  let invalid = false;
+  let score = 0;
+  let bonus = 100;
+
+  if (arr === undefined){
+    return -1;
+  }
+  if (!Array.isArray(arr)){
+    return undefined;
+  }
+  arr.forEach(function(radius){
+    if (radius < 1 || radius > 20){
+      invalid = true;
+    }
+  })
+  if (invalid){
+    return undefined;
+  }
+
+  arr.forEach(function(radius){
+    if (radius > 5){
```

Comment:
I think you got the math here wrong. Take a deeper look into the README and see what the scores should be.